cli/foundation-models/system/inference/summarization/summarization-batch-endpoint.sh

```diff
@@ -0,0 +1,96 @@
+set -x
+# the commands in this file map to steps in this notebook: https://aka.ms/azureml-infer-batch-sdk-summarization
+
+# script inputs
+registry_name="azureml"
+subscription_id="<SUBSCRIPTION_ID>"
+resource_group_name="<RESOURCE_GROUP>"
+workspace_name="<WORKSPACE_NAME>"
+
+# This is the model from system registry that needs to be deployed
+model_name="sshleifer-distilbart-cnn-12-6"
+
+# Validate the existence of the model in the registry and get the latest version
+model_list=$(az ml model list --name ${model_name} --registry-name ${registry_name} 2>&1)
+if [[ ${model_list} == *"[]"* ]]; then
+    echo "Model doesn't exist in registry. Check the model list and try again."; exit 1;
+fi
+version_temp=${model_list#*\"version\": \"}
+model_version=${version_temp%%\"*}
+
+version=$(date +%s)
+endpoint_name="summarization-$version"
+job_name="summarization-job-$version"
+
+# todo: fetch compute_sku from the min_inference_sku tag of the model
+compute_sku="Standard_DS5_v2"
```

Comment:
does this model require a larger sku?

Reply:
For online endpoints, it works with DS4_v2.